src/ucp/core/ucp_ep.c

```diff
@@ -2880,7 +2880,14 @@ void ucp_ep_vfs_init(ucp_ep_h ep)
 {
     ucp_err_handling_mode_t err_mode;
 
+#if ENABLE_DEBUG_DATA
+    ucs_vfs_obj_add_dir(ep->worker, ep, "ep/%s", ep->name);
```

Comment:
what happens in case of name collision?

Reply:
will merge it now, next PR could add name conflict resolve for context,worker,ep,rcache

Reply:
Ok, thanks.